src/java.base/share/classes/java/io/FileReader.java

```diff
@@ -47,40 +47,40 @@
 
    /**
     * Creates a new {@code FileReader}, given the name of the file to read,
-    * using the platform's
-    * {@linkplain java.nio.charset.Charset#defaultCharset() default charset}.
+    * using the {@linkplain java.nio.charset.Charset#defaultCharset() default charset}.
```

Comment:
The java.io classes import java.nio.charset.Charset so don't need to use the fully qualified class name everywhere.

Reply:
Corrected, as well as other locations.